src/ResourceManagement/SqlManagement/Sql.Tests/ImportExportScenarioTests.cs

```diff
@@ -127,24 +127,6 @@ public void TestImportExport(bool preexistingDatabase, string testPrefix, string
                         MaxSizeBytes = (2 * 1024L * 1024L * 1024L).ToString(),
                     });
                 }
-
-                // List Server import/export operations
```

Comment:
@jaredmoo is this removed block being tested somewhere else? Making sure there isn't any test regression with this being removed